crypto/ethsecp256k1/ethsecp256k1.go

```diff
@@ -60,15 +60,15 @@ func (privKey *PrivKey) Bytes() []byte {
 }
 
 // PubKey returns the ECDSA private key's public key.
-func (privKey PrivKey) PubKey() tmcrypto.PubKey {
+func (privKey PrivKey) PubKey() cryptotypes.PubKey {
 	ecdsaPrivKey := privKey.ToECDSA()
 	return &PubKey{
 		Key: ethcrypto.CompressPubkey(&ecdsaPrivKey.PublicKey),
 	}
 }
 
 // Equals returns true if two ECDSA private keys are equal and false otherwise.
-func (privKey *PrivKey) Equals(other tmcrypto.PrivKey) bool {
+func (privKey *PrivKey) Equals(other cryptotypes.LedgerPrivKey) bool {
```

Comment:
```suggestion
func (privKey *PrivKey) Equals(other cryptotypes.PrivKey) bool {
```

Reply:
Can't be replaced, because the interface `cryptotypes.PrivKey` defines the function `Equals` with `cryptotypes.LedgerPrivKey`.